addons/sys_core/fnc_speaking.sqf

```diff
@@ -100,7 +100,7 @@ if !(GVAR(keyedMicRadios) isEqualTo []) then {
     private _compiledParams = HASH_CREATE;
     {
         private _recRadio = _x;
-        if (_recRadio != ACRE_BROADCASTING_RADIOID || {GVAR(fullDuplex)}) then {
+        if (_recRadio != ACRE_BROADCASTING_RADIOID || {GVAR(fullDuplex)} || toLower(_recRadio) in ACRE_SPECTATOR_RADIOS) then {
```

Comment:
```suggestion
        if (_recRadio != ACRE_BROADCASTING_RADIOID || {GVAR(fullDuplex)} || {toLower(_recRadio) in ACRE_SPECTATOR_RADIOS}) then {
```

Reply:
At least that broke my tests yesterday. So there are still leftovers where we have mixed cases

Reply:
Scratch that